libs/ngx-dsfr/checkbox/src/lib/checkbox.component.html

```diff
@@ -3,10 +3,11 @@
     <input
       [checked]="isChecked(items[0].value)"
       (change)="onInputChange($event)"
+      [attr.aria-describedby]="ariaAttribute"
```

Comment:
Given what's in the doc, you should handle the value for `aria-describedby` and `aria-labelledby` differently : 
- `aria-describedby` expects only one ID
- `aria-labelledby` expects 2 IDs

Which makes sense since, for a single checkbox, there is no `fieldset` and no `legend`...

Reply:
Done